from exception import except_sample as error_example

# Press the green button in the gutter to run the script.
if __name__ == '__main__':
    error_example.try_except_finally()

# See PyCharm help at https://www.jetbrains.com/help/pycharm/
